src/openrct2/platform/Platform.Android.cpp

```diff
@@ -44,6 +44,16 @@ namespace Platform
         Guard::Assert(false, "GetCurrentExecutablePath() not implemented for Android.");
         return std::string();
     }
+
+    uintptr_t StrDecompToPrecomp(utf8* input)
+    {
+        return platform_str_decomp_to_precomp(input);
```

Comment:
Do not redirect to a C-style function. Either handle it directly or create a namespaced C++ style helper function.

Reply:
All of the `Platform.*`  are implementing the functions declared in ´Platform2.h`. but I didn't want to replicate the same code for instances where there was no special handling (such as everything that is not a MacOS here), so I made them call things in `Shared`

Reply:
Yes, but this function is so trivial that redirecting is not useful. And even if it was, it still shouldn't redirect to a C-style function, but to a C++-style one.

In any case: please replace the redirection with the actual implementation here.

Reply:
Done.